client/src/nv_ingest_client/primitives/tasks/store.py

```diff
@@ -49,7 +64,8 @@ def __init__(
         structured: bool = True,
         images: bool = False,
         store_method: _Type_Store_Method = None,
-        **extra_params,
+        extra_params: dict = None,
```

Comment:
this change is critical, unknown keyword arguments are not passed to the task when created via the client. So when you try to use the minio credentials to access a private bucket by getting them from the params dictionary you end up not finding anything so you default to `none`. This change makes extra_params an explicit argument and therefore carries over when the task is recreated via the service. It also leaves params available because that is used by the CLI tool to blanket pass information to tasks.

Reply:
Also we can use keyword arguments directly because when we go to instantiate using the Ingest schema it complains that were are passing keyword arguments that are not expressly defined in the schema because we explicitly forbid that in the ingest schemas. Therefore, I decided to keep that standard in place and use the extra_params dictionary to capture all necessary connection information, this coincides with how other libraries also handle connection parameters (i.e. pandas, s3fs).